src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ERR.cs

```diff
@@ -21,14 +21,16 @@ internal static partial class Crypto
         internal static extern IntPtr ErrReasonErrorString(ulong error);
 
         [DllImport(Libraries.CryptoNative, CharSet = CharSet.Ansi)]
-        private static extern void ErrErrorStringN(ulong e, [Out] StringBuilder buf, int len);
+        private static unsafe extern void ErrErrorStringN(ulong e, byte* buf, int len);
```

Comment:
Why not just take byte[] here and pass the byte[] in below?  That way you don't need to use unsafe code at all.  Or is Marshal.PtrToStringAnsi more performant than Encoding.Default.GetString?